src/tutorials/selfhosting/administration/more_instances.md

```diff
@@ -0,0 +1,90 @@
+# Hosting more than one Cozy instance on the same server
+
+Having its own selfhosted Cozy instance is nice but hosting Cozy instances for friends and family is a must! Here is how to add more Cozy instances on the same server.
+
+The first Cozy instance we added was `https://cozy.domain.example`. We will create a second Cozy instance for Mary with address `https://mary.domain.example` (Replace `domain.example` with your own domain name and `mary` with what you want to uniquely identify the Cozy instance.
+
+So we will need:
+
+- Our domain name. We still use `domain.example` in this documentation
+- The new Cozy instance's “slug”, which is its unique identifier. We will use `mary` here for example. The address for this new Cozy instance will the be in the form `https://<slug>.<domain>`, for example here `https://mary.domain.example`
+
+First, let's put all that important information in variables:
+
+    DOMAIN=domain.example
+    EMAIL=<your email addresse>
+    NEWSLUG=mary
+    NEWEMAIL=<Mary's email address>
+
+Create DNS entries for this Cozy instance. For example:
+
+    mary     1h     IN         A     <your_server_IP>
+    *.mary   1h     IN     CNAME     mary
+
+Create Nginx base configuration for this Cozy instance:
+
+    cat <<EOF | sudo tee /etc/nginx/sites-available/${NEWSLUG}.${DOMAIN} > /dev/null
+    server {
+        listen 80;
+        listen [::]:80;
+
+        root /var/www/html;
+        server_name *.${NEWSLUG}.${DOMAIN} ${NEWSLUG}.${DOMAIN};
+
+        location /.well-known {
+            alias /var/www/html/.well-known;
+        }
+
+        location / {
+            return         301 https://\$host\$request_uri;
+        }
+    }
+    EOF
+    sudo ln -s ../sites-available/${NEWSLUG}.${DOMAIN} /etc/nginx/sites-enabled/
+    sudo systemctl reload nginx
+
+Generate SSL certificate using certbot:
+
+    sudo certbot certonly --email "${EMAIL}" --non-interactive --agree-tos --webroot -w /var/www/html -d ${NEWSLUG}.${DOMAIN} $(printf -- " -d %s.${NEWSLUG}.${DOMAIN}" home banks contacts drive notes passwords photos settings store)
```

Comment:
Should we add `coachco2` & `mespapiers`?

Reply:
mespapiers should definitely be there (Don't know why I lost it, probably I started from master before #266 merge)

I'm adding it.

CoachCO2 is not available to self hosted nor cozy-hosted users atm, it's an application only available to business clients.